p2p/receiver/validator.go

```diff
@@ -105,30 +101,6 @@ func HandshakeValidator(networkID uint32, idService identity.Service) ValidatorG
 	}
 }
 
-// DocumentAccessValidator validates the GetDocument request against the AccessType indicated in the request
-func DocumentAccessValidator(doc *coredocumentpb.CoreDocument, docReq *p2ppb.GetDocumentRequest, requesterCentID identity.CentID) error {
```

Comment:
I think is better to comment this out knowing that we will need this soon. Or has it been moved?

Reply:
It has been moved to the CoreDocModel.

Reply:
ok